src/Remind/GenericRemindService.cs

```diff
@@ -38,7 +38,7 @@ public async Task<string> SendReminderAsync(IBotFrameworkHttpAdapter adapter)
             List<UserProfile> userProfiles = await _userProfilesProvider.GetUserProfilesAsync();
 
             List<UserProfile> userToRemind = userProfiles
-                .Where(u => u.ClockifyToken != null && u.ConversationReference != null)
+                .Where(u => u.ClockifyTokenId != null && u.ConversationReference != null)
```

Comment:
this is annoying, as we're likely to skip a few reminder for everyone.
Can we avoid it as a temporary solution (to be removed once we get rid of the field entirely)?

Reply:
I checked, we don't really need the token, it was a smart move to avoid users who never signed in
So we can change it to

`.Where(u => (u.ClockifyTokenId ?? u.ClockifyToken) != null && u.ConversationReference != null)`